doc/source/whatsnew/v1.4.3.rst

```diff
@@ -30,6 +30,7 @@ Fixed regressions
 - Fixed regression in :func:`assert_index_equal` when ``check_order=False`` and :class:`Index` has extension or object dtype (:issue:`47207`)
 - Fixed regression in :func:`read_excel` returning ints as floats on certain input sheets (:issue:`46988`)
 - Fixed regression in :meth:`DataFrame.shift` when ``axis`` is ``columns`` and ``fill_value`` is absent, ``freq`` is ignored (:issue:`47039`)
+- Fixed regression in :meth:`DataFrame.to_json` when ``index`` is of the type ``PeriodIndex`` causing a segmentation violation (:issue:`46683`)
```

Comment:
:class:`Index` and class:`PeriodIndex`

Reply:
`index` refers to the parameter in the constructor of `DataFrame` and not the class. Is there some kind of markup to indicate parameters?
The `PeriodIndex` has been marked as class

Reply:
for parameter types then the double ticks is fine. However, the note refers to `to_json` so it is not obvious that the `index` is a parameter name. perhaps reword to avoid.

Reply:
I see the confusion. Is the new version clearer?

Reply:
lgtm.

Reply:
I think marking this as class too is clearer, since both are interchangable in this context. Otherwise, please add something like when argument ``ìndex``  is ...

Reply:
Fixed